core/src/test/resources/stacks/tcp.xml

```diff
@@ -1,26 +1,27 @@
 <config xmlns="urn:org:jgroups"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-        xsi:schemaLocation="urn:org:jgroups http://www.jgroups.org/schema/JGroups-3.4.xsd">
-   <TCP
+        xsi:schemaLocation="urn:org:jgroups http://www.jgroups.org/schema/JGroups-3.6.xsd">
+   <!-- Disable the regular thread pool queue for tests so we can set a lower min_threads -->
+   <TCP_NIO2
          bind_addr="${jgroups.tcp.address:127.0.0.1}"
          bind_port="${jgroups.tcp.port:7800}"
          port_range="30"
-         recv_buf_size="20m"
+         recv_buf_size="2m"
          send_buf_size="640k"
          max_bundle_size="31k"
          use_send_queues="true"
          enable_diagnostics="false"
          tcp_nodelay="true"
-         bundler_type="sender-sends-with-timer"
+         bundler_type="transfer-queue"
```

Comment:
Oops, I need to revisit these config changes... I had the TCP_NIO2 changes stashed in a local branch for a long time :)